js/react_native/test_types_models.readme.md

```diff
@@ -0,0 +1,15 @@
+`js/react_native/android/src/androidTest/res/raw/test_types_*.ort` and
```

Comment:
>js/react_native/android/src/androidTest/res/raw/test_types_*.ort` and [](http://example.com/codeflow?start=1&length=69)

Given the fact that .ort format is platform and version specific, does it make sense to keep this in source control and maintain it as it is now instead of generating on the fly?

Reply:
the models should not be platform specific. admittedly, it is a pain to regenerate everything manually, but the model format is normally expected to be stable and backwards compatible.

it would be nice to generate them on the fly, but that requires a certain amount of infrastructure to do. namely, a full build with Python.